webui/src/lib/components/repository/changes.jsx

```diff
@@ -143,7 +143,7 @@ export const TreeEntryRow = ({entry, relativeTo = "", leaf = false, dirExpanded,
         pathSection = <Link href={onNavigate(entry)}>{pathSection}</Link>
     }
     const rowActions = []
-    if (onClickExpandDiff) {
+    if (onClickExpandDiff && entry.type !== 'conflict') {
```

Comment:
You don't have to but consider exporting this and other diff types to the new constants.jsx file